components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.v1/src/main/java/org/wso2/carbon/identity/api/server/secret/management/v1/core/SecretManagementService.java

```diff
@@ -179,8 +182,9 @@ public SecretResponse getSecret(String secretType, String name) {
     /**
      * To make a partial update or update the specific property of the secret.
      *
-     * @param name               secret name.
-     * @param secretPatchRequest secret's patch details.
+     * @param secretType Secret type name.
```

Comment:
Description indentation not consistent with the params/descriptions formatting below.